v2/azure-arm.yaml

```diff
@@ -1782,6 +1818,12 @@ objectModelConfiguration:
         Id:
           $armReference: true
     2024-04-02-preview:
+      ContainerServiceSshPublicKey:
```

Comment:
Just a thought - can we exclude `PublicKey` from the secret check heuristic as I think we might encounter these in the future too.

Reply:
I see below its already marked as `IsSecret: False` in the `SecretDetector`. So we can remove these now?

Reply:
The property `KeyData` is triggering the heuristic - we don't currently use the object's name. Though maybe we should. 🤔

Reply:
I'll think about doing this as a followup.